### DISCUSSION TITLE
DNSBlockApp

### DISCUSSION THREAD

**Discussion by WEBDIENSTE** [TechnitiumSoftware/DnsServer#1081] [Q&A]
Hi!

I have a few questions about the DNSBL App.

I'd like to create my own DNSBL service for my mail servers.
Therefore I'd like to import a few external lists (URLs), stored on my own webhosting.
Is it possible to add multiple external links or must the file/a be stored locally?
Is it possible to add more than one local file and where is the directory where the file should be stored?

Thank you!

**Comment by ShreyasZare** [comment]
Thanks for asking. The files are expected to be locally stored and you cannot use URL for it. You can configure the app to use more than one file by creating multiple json object blocks, one for each file. So, you do not need to combine those multiple files and you can keep them anywhere you wish on the disk.

You can update the files my any mechanism and the app will detect that using the last modified time stamp for the file and reload it.
